ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1578,6 +1578,53 @@ literals:
 - valve_status:
   - OPEN
   - CLOSED
+
+- failed_panel_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_smoke_alarm: # for aspirating type smoke detectors
+  - ACTIVE
+  - INACTIVE
+- medium_smoke_alarm: # for aspirating type smoke detectors
+  - ACTIVE
+  - INACTIVE
+- particle_concentration_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_battery_alarm:
+  - ACTIVE
+  - INACTIVE
+- load_bypass_status:
+  - OFF
+  - ON
+- use_battery_status:
+  - OFF
+  - ON
+- gas_release_status:
+  - OPEN
+  - CLOSED
+- panel_controller_status:
+  - NORMAL
+  - UNKNOWN
+- motorized_valve_position_command:
+  - CLOSED
+  - OPEN
+- motorized_valve_position_status:
+  - CLOSED
+  - OPEN
+- motorized_damper_position_command:
```

Comment:
same as above, replace with `damper_command` and `damper_status`. the status already exists, so you only need to add the command.

Reply:
noted sir. Thanks